README.Rmd

```diff
@@ -30,3 +30,30 @@ knitr::opts_chunk$set(
 * Whose run-time is not practical for individual packages. 
 
 These tests are run on a cron job and are run for both CRAN versions and the current GitHub development versions. 
+
+
+## PR-pairs with package repos
+
+PRs on extratests typically are part of a PR pair since they test changes in package repositories. The following workflow ensures that the CI run triggered by the PR on extratests runs with the changes in the corresponding PR on the package repository.
+
+Normal development
+* [pkg] Make changes
```

Comment:
```suggestion

* [pkg] Make changes
```

In the [branch's preview](https://github.com/tidymodels/extratests/tree/bb302ac6816c318c82c6b1b7f9f179852f9d6f42#pr-pairs-with-package-repos), I see that Markdown doesn't pick up on the bulleted list format. I think that's because of missing spaces before bullets begin.